### PR TITLE
feat: support ConfigProvider card bordered

### PR DESCRIPTION
**Pull request by yee94**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [x] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 💡 Background and solution

Card 组件的 `bordered` 是由 Props 进行控制，部分设计规范中，对于 Card 是不需要 border 的，为了满足此类需求，需要在 ConfigProvider 中添加全局控制。对已有使用场景不受影响。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     Support card bordered config in `ConfigProvider`      |
| 🇨🇳 Chinese |     在`Config Provider`中支持卡边框配置      |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
Please send this PR to feature branch.

File: components/card/Card.tsx [hunk 1]
```diff
@@ -103,7 +103,6 @@ const Card = React.forwardRef<HTMLDivElement, CardProps>((props, ref) => {
     bodyStyle = {},
     title,
     loading,
-    bordered = true,
```
Comment (crazyair) [inline components/card/Card.tsx:106]: `bordered = cpCard.bordered ?? true`

Comment (yee94) [reply components/card/Card.tsx:106]: OK，按照这种写法改一下

Comment (crazyair) [reply components/card/Card.tsx:106]: 现在初始化了2个 cp，合并下吧

**Comment by crazyair** [comment]
新功能，使用 feture 分支

**Comment by yee94** [comment]
> 新功能，使用 feature 分支

https://github.com/ant-design/ant-design/pull/47818

**Comment by Wxh16144** [comment]
> > 新功能，使用 feature 分支
> 
> #47818

ohh，应该在这个 pr 上改目标分支为 feature就好了， 至于你的分支，可以切一个作为备份，然后 rebase 一下，force push 就好了。

这样的好处是这个 pr 的 review 回复可以保留在一个pr 上。而不用切换到另外一个 pr

**Pull request closed without merging**